docs/server-request.md

```diff
@@ -17,20 +17,38 @@ A low-level action that allows for any kind of HTTP request to be performed.
 - Read files or binary streams
 - Call APIs from same app system - use the [Call Local API](/docs/Actions/CallLocalApi.md) action instead
 
-## `Related Actions`
+## `Related actions`
 
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
 
-## `Parameter Reference`
+## `Parameter reference`
 
-| Parameter     | Description                           | Supports Variables |
+| Parameter     | Description                           | Supports variables |
 |---------------|---------------------------------------|--------------------|
 | URL           | Absolute URL where the resource to be invoked lives. | Yes |
-| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD. | No |
+| Enforce SSL   | This will for ce request to be sent to a secure version of the URL.(https instead of http)| No |
+| Timeout       | The amount of time in which the response must come or the request fails. | Yes |
+| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD, PATCH. | No |
 | Data          | If the HTTP Method is POST or PUT, data is allowed to be sent to the target URL. The data can take various forms, from key-value pairs to complex JSON. For the target URL to understand the format, it needs to be passed through the Content-Type header. | Yes |
+| Headers | Additional headers to pass with the request. | Yes |
+| Ignore Errors | Any errors thrown when executing this action will be ignored. Using this options will not stop the triggering of the 'On Error' event. | No |
```

Comment:
Just to confirm, the action will execute the OnError actions after which it will allow the execution to continue with the next action instead of showing an error?

Reply:
Anytime an exception is thrown if we have OnError actions they will be executed. IF ignore errors is checked, we will just log the exception and continue silently.